tv.kodi.Kodi.metainfo.xml

```diff
@@ -32,7 +32,42 @@
     </description>
     <launchable type="desktop-id">tv.kodi.Kodi.desktop</launchable>
     <screenshots>
-        <screenshot type="default">https://cdn.dday.it/system/uploads/news/main_image/22334/main_a1.jpg</screenshot>
+        <screenshot type="default">
+            <image>https://mirrors.kodi.tv/screenshots/kodi-recently-added.jpg</image>
+            <caption>The homescreen keeps your media organized</caption>
+        </screenshot>
+        <screenshot type="default">
```

Comment:
Why do you have multiple default screenshots?

Reply:
According to the spec, this shouldn't be a problem, but your right, it's an oversight on my part.

Reply:
Removed this from upstream. Will change this MR to draft, till that's merged.
https://github.com/xbmc/xbmc/pull/20142